### ISSUE TITLE
Search - Returns objects that does not match

### ISSUE THREAD

**Issue by anderserla** [Sundsvallskommun/OpenEMap-WebUserInterface#165]
Search returns hits that dont really match. Do the following
1. Search for "130" -> Error there are 10 hits, but it should only be 1

This may relate to the ranking of hits. We would only want exact matches. Maybe it also searches the coordinates? Any suggestions @sweco-sebhar ?

**Comment by sweco-sebhar** [comment]
Yes, I think coordinates have been indexed when loaded into ElasticSearch. We do use the geometry as returned by ElasticSearch so it cannot be removed but it should be possible to configure ElasticSearch to not index it. Probably a configuration that needs to be done when loading data.
